_posts/2020-02-14-status-update.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: "14th Feb 2020: IAC Team Status Update"
+author: criskoduck
+categories:
+  - team
+  - status
+tags:
+  - module updates
+  - tooling
+  - personal
+  - honeycomb
+  - metrics
+  - litmus
+  - windows
+  - powershell
+---
+
+The IAC team have been working on a lot of interesting things these past two weeks. Here's a brief overview of what we've been up to:
+
+## Updates to Puppet Modules
```

Comment:
Technically we released two! pwshlib went out last night with a bugfix for puppet server 5.x

Reply:
Cool - I'll add it. Kerminator didn't list it, unfortunately. 11 hours ago - must be below the cut off?